topi/include/topi/scale.h

```diff
@@ -0,0 +1,286 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file topi/scale.h
```

Comment:
move into topi/image/resize.h, as name scale is a bit confusing.

Reply:
Moved resize under new category "image" across nnvm & tvm